SimpleUploadServer/index.js

```diff
@@ -64,6 +64,8 @@ const server = new ApolloServer({
   }
 });
 
-server.listen().then(({ url }) => {
+server.listen({
+  port: 4001
```

Comment:
The Apollo Server documentation example defaults to port `4000`. Since it's maintained by other teams and serves as a documentation example it's easier for us to move the SimpleUploadServer service to port `4001`.